roles/setup_assisted_installer/defaults/main.yml

```diff
@@ -9,6 +9,7 @@ assisted_service_image: "quay.io/ocpmetal/assisted-service:{{ assisted_service_t
 assisted_service_gui_image: "quay.io/ocpmetal/ocp-metal-ui:{{ assisted_service_gui_tag }}"
 assisted_postgres_image: quay.io/ocpmetal/postgresql-12-centos7
 
+# Don't forget to update the sample inventory
```

Comment:
I am thinking that all the comments in the inventory should get removed and replaced with a step by step guide to create the inventory from scratch.

Reply:
I'm not sure. I like the comments in the inventory as they are in the place of use if that makes sense. 
We can discuss it with the rest of the team.

Reply:
I think leaving comments in the inventory is fine, although repeating the comment may be unnecessary here.

As soon as we wrap Crucible with something to simplify the UX of the interface (say a bash script or a python script) to get rid of the ansible-playbook command requiring multiple params, Ansible then becomes an implementation detail, something that is run in the background to do the heavy lifting throughout the deployment process.

Making things easier for people who don't have a lot of experience with Ansible by adding a sentence or two to the docs doesn't cost much, and facilitates transforming the goal from  "I want an OpenShift cluster deployed using Ansible" to "I want an OpenShift cluster deployed", which is an ultimately better goal. At least in my view :P

What I'd suggest is to remove this comment from the snippet as it's repeated multiple times, and instead, leave a note above the snippet explaining when and what `ansible_connection: local` is used for.